lib/python/rose/config.py

```diff
@@ -1138,6 +1139,16 @@ def _comment_format(cls, comment):
         """Return text representation of a configuration comment."""
         return "#%s\n" % (comment)
 
+    @staticmethod
```

Comment:
Prize for the best name for this method - one hand-folded origami rice paper eating hat.